scenarios/appsec/test_runtime_activation.py

```diff
@@ -0,0 +1,33 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import BaseTestCase, context, coverage, interfaces, released, irrelevant
+
+# dd.rc.targets.key.id=TEST_KEY_ID
+# dd.rc.targets.key=1def0961206a759b09ccdf2e622be20edf6e27141070e7b164b7e16e96cf402c
+# private key: a78bd01afe0dc0baa6904e1b65448a6bbe160e07f7fc375c3bcb3ec08f008cc5
+
+
+@released(java="0.115.0", cpp="?", dotnet="2.16.0", php="?", python="?", ruby="?", nodejs="?", golang="?")
+@irrelevant(context.appsec_rules_file == "")
```

Comment:
I'm not sure to understnad the purpose here ?

Reply:
If there are custom rules, AppSec's behavior is not updating them through remote config. So this scenario cannot be run properly with custom rules.

Reply:
The point is that all scenario should run only relevant test classes. If a scenario uses a custom rules set, then it should not include this test class.

Reply:
@cbeauchesne what controls whether the condition is true or not is not the scenario being run, but rather the content of the `binaries/` directory. If there is a custom rules file there, the test will fail.

Reply:
You're perfectly right !